### DISCUSSION TITLE
Transparent Avatar - Dark mode

### DISCUSSION THREAD

**Discussion by HPaulson** [community/community#2865] [Other Feature Feedback, Questions, & Ideas] [answered]
Currently, for Organizations, Avatar images cannot be transparent. When uploading a transparent image, GitHub will still display the image with a white background. I'm sure this issue stems from not allowing png filetypes. This used to be no problem, however, due to the new dark mode, this looks extremely bad in some cases. It would be nice to have this image be transparent, this way on the dark mode it does not display with the very-ugly white background. 

Example:
![](https://cdn.seismiccore.com/axe.png)

**Comment by ash-development** [comment, marked as answer]
not github staff, but this is not the place for appearance feedback. you need to go to [https://github.community/c/github-help/dark-mode-beta/65](https://github.community/c/github-help/dark-mode-beta/65)

**Reply by evi-liu** [reply]
GitHub staff here, and this is correct. @HPaulson do you mind transferring this feedback to https://github.community/c/github-help/dark-mode-beta/65?

**Reply by HPaulson** [reply]
@evi-liu @ty-schnoor I absolutely will, very sorry about that!

**Reply by evi-liu** [reply]
no worries! there are a lot of different places to post feedback for GitHub so i understand the confusion 😅

**Reply by mrquincle** [reply]
For the convenience of future visitors. This conversation has been continued in https://github.community/t/transparent-avatar-dark-mode/148628 and then in https://github.community/t/white-backgrounds-for-user-and-organization-images/147902

**Reply by tisonkun** [reply]
It seems after GitHub moves to Discussions, the link gets above broken.
